src/cpp/session/modules/SessionHelp.cpp

```diff
@@ -125,6 +125,21 @@ bool isLocalURL(const std::string& url,
    return false;
 }
 
+std::string normalizeHttpdSearchContent(const std::string& content)
+{
+   return boost::regex_replace(
+            content,
+            boost::regex("(The search string was <b>\")(.*)(\"</b>)"),
+            [](const boost::smatch& m)
+   {
+      std::string query = m[2];
+      if (query.find('<') != std::string::npos)
+         query = string_utils::htmlEscape(query);
+
+      return m[1] + string_utils::htmlEscape(m[2]) + m[3];
```

Comment:
Of course, with that in mind we should be using `query` here rather than `m[2]`!